From a5d54403cad886ff73a3051d6c3d2fb796e1f65b Mon Sep 17 00:00:00 2001
From: Charles Short <charles.short@windriver.com>
Date: Wed, 12 Jan 2022 11:42:20 -0500
Subject: [PATCH] Add /etc/etcd configuration directory

Ansible playbook expects an /etc/etcd directory so
create one when the etcd-server is installed.

Signed-off-by: Charles Short <charles.short@windriver.com>
---
 debian/etcd-server.postinst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/etcd-server.postinst b/debian/etcd-server.postinst
index 0bdc5c20..3ed3754e 100644
--- a/debian/etcd-server.postinst
+++ b/debian/etcd-server.postinst
@@ -8,6 +8,7 @@ case $1 in
             adduser --system --group --disabled-login --disabled-password --home /var/lib/etcd/ etcd
         fi
         chmod 700 /var/lib/etcd/
+	 mkdir -m 755 -p /etc/etcd
     ;;
     abort-upgrade|abort-remove|abort-deconfigure)
     ;;
-- 
2.25.1

